build.gradle.kts

```diff
@@ -27,3 +29,23 @@ allprojects {
         }
     }
 }
+
+subprojects {
+    tasks.withType<KotlinCompile>().all {
+        kotlinOptions {
+            if (project.findProperty("enableComposeCompilerReports") == "true") {
+                freeCompilerArgs += listOf(
+                    "-P",
+                    "plugin:androidx.compose.compiler.plugins.kotlin:reportsDestination=" +
+                            project.buildDir.absolutePath + "/compose_metrics"
```

Comment:
Not safe path builder. File separators depend on OS.
Probably does not handles file path with spaces.

```kotlin
freeCompilerArgs += listOf(
  "-P",
  "plugin:androidx.compose.compiler.plugins.kotlin:reportsDestination=\"${File(project.buildDir, "compose_metrics")}\""
)
```

Reply:
Not sure about `absolutePath`, probably relational path to the project folder will be enough, but I am fine with absolute too.